packages/orbit-components/src/NavigationBar/index.tsx

```diff
@@ -64,12 +70,17 @@ const NavigationBar = ({
       data-test={dataTest}
       id={id}
       className={cx(
-        "bg-white-normal p-300 z-navigation-bar fixed inset-x-0 top-0 box-border flex w-full translate-x-0 items-center",
-        "duration-normal transform-gpu transition-transform ease-in-out",
+        "p-300 z-navigation-bar fixed inset-x-0 top-0 box-border flex w-full translate-x-0 items-center",
+        "duration-normal transform-gpu ease-in-out",
+        transparentBgAtTop
+          ? "transition-[transform,background-color,border-color]"
+          : "transition-transform",
         "tb:h-1600 h-[52px]", // As defined on the const above
-        shown ? "translate-y-0" : "tb:-translate-y-1600 translate-y-[-52px]", // As defined on the const above
-        bottomStyle === "shadow" && "shadow-fixed",
-        bottomStyle === "border" && "border-cloud-normal border-b",
+        shown ? "translate-y-0" : "tb:-translate-y-1600 -translate-y-xxxl", // As defined on the const above
```

Comment:
xxxl is deprecated, so we should keep the hardcoded value

Reply:
It was the auto-formatting that did this, I tried many times to bypass it and looks like I failed in my last push 😅

Reply:
We need to get rid of that long deprecated xxxl token asap 😌